### PR TITLE
pkg: include lite in PL

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

until Lite is released we shall include it the PL pckage

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @carmocca @akihironitta

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
🎉 ❤️

File: src/pytorch_lightning/__setup__.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def _adjust_manifest(**__: Any) -> None:
     lines += [
         "recursive-exclude src *.md" + os.linesep,
         "recursive-exclude requirements *.txt" + os.linesep,
+        "recursive-include requirements/lite *.txt" + os.linesep,  # TODO: remove lite after fist LLite release
+        "recursive-include src/lightning_lite *.md" + os.linesep,  # TODO: remove lite after fist LLite release
```
Comment (carmocca) [inline src/pytorch_lightning/__setup__.py:57]:
```suggestion
        "recursive-include requirements/lite *.txt" + os.linesep,  # TODO: remove after the first standalone Lite release
        "recursive-include src/lightning_lite *.md" + os.linesep,   # TODO: remove after the first standalone Lite release
```

File: src/pytorch_lightning/__setup__.py [hunk 2]
```diff
@@ -78,7 +80,15 @@ def _setup_args(**__: Any) -> Dict[str, Any]:
...
         download_url="https://github.com/Lightning-AI/lightning",
         license=_about.__license__,
-        packages=find_packages(where="src", include=["pytorch_lightning", "pytorch_lightning.*"]),
+        packages=find_packages(
+            where="src",
+            include=[
+                "pytorch_lightning",
+                "pytorch_lightning.*",
+                "lightning_lite",  # TODO: remove lite after fist LLite release
+                "lightning_lite.*",  # TODO: remove lite after fist LLite release
```
Comment (carmocca) [inline src/pytorch_lightning/__setup__.py:89]:
```suggestion
                "lightning_lite",  # TODO: remove after the first standalone Lite release
                "lightning_lite.*",  # TODO: remove after the first standalone Lite release
```

**Review by carmocca — APPROVED** [review]
How can we tell that this worked?

**Comment by Borda** [comment]
> How can we tell that this worked?

Yes, need to finish the CI update...

**Comment by awaelchli** [comment]
@Borda I updated a check that was failing:

```
Run cp dist/* pypi/
Run for file in `ls *.gz`; do tar -xzf $file; done
Run import os, glob, pathlib, shutil
['pytorch-lightning-1.8.0.dev0/src/pytorch_lightning', 'pytorch-lightning-1.8.0.dev0/src/lightning_lite']
Traceback (most recent call last):
  File "/home/runner/work/_temp/8930d45f-d606-482d-b5cd-9a42114d21e1.py", line 5, in <module>
    assert len(dirs) == 1
AssertionError
Error: Process completed with exit code 1.
```

**Comment by Borda** [comment]
> But then it doesn't work with the meta package

yes, that does ot work of any other package :)

**Pull request merged**